supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-enterprise-linux.adoc

```diff
@@ -67,3 +67,834 @@ For documentation questions, contact rhel-docs@redhat.com.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*: xref:ansible-playbook[ansible-playbook]
+
+[discrete]
+[[ansible-task]]
+==== image:images/yes.png[yes] Ansible task (noun)
+*Description*: Ansible tasks are units of action in Ansible. An Ansible play can contain multiple tasks. The goal of each task is to execute a module, with very specific arguments.
+An Ansible task is a set of instructions to achieve a state defined, in its broad terms, by a specific Ansible role or module, and fine-tuned by the variables of that role or module.
+For more information, see the link:https://docs.ansible.com/ansible/latest/user_guide/basic_concepts.html#tasks[official Ansible tasks documentation].
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[apache-web-server]]
+==== image:images/yes.png[yes] Apache web server (noun)
+*Description*: The Apache HTTP Server, colloquially called Apache, is a free and open-source cross-platform web server application, released under the terms of Apache License 2.0. Apache played a key role in the initial growth of the World Wide Web, and is currently the leading HTTP server. Its process name is `httpd`, which is short for _HTTP daemon_. Red Hat Identity Management (IdM) uses the Apache Web Server to display the IdM Web UI, and to coordinate communication between components, such as the Directory Server and the Certificate Authority.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*: xref:certificate[certificate], xref:certificate-authorities[certificate-authorities], xref:directory-server[directory-server]
+
+[discrete]
+[[blueprint]]
+==== image:images/yes.png[yes] blueprint (noun)
+*Description*: Blueprints are simple text files in Tom's Obvious, Minimal Language (TOML) format that describe which packages, and what versions, to install into the image. They can also define a limited set of customizations to make to the final image.
+
+*Use it*: yes
+
+*Incorrect forms*: blue print, BluePrint
+
+*See also*:
+
+[discrete]
+[[certificate]]
+==== image:images/yes.png[yes] certificate (noun)
+*Description*: A certificate is an electronic document used to identify an individual, a server, a company, or other entity and to associate that identity with a public key. Such as a driver's license or passport, a certificate provides generally recognized proof of a person's identity. Public-key cryptography uses certificates to address the problem of impersonation.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*: xref:certificate-authorities[certificate-authorities]
+
+[discrete]
+[[certificate-authorities]]
+==== image:images/yes.png[yes] Certificate Authorities (noun)
+*Description*: An entity that issues digital certificates. In {RH} {IPA}, the primary CA is `ipa`, the IdM CA. The `ipa` CA certificate is one of the following types:
```

Comment:
{RH} {IPA} - This uses attributes that aren't in the SSG. Can you hard code whatever these values are supposed to be?